test/hotspot/jtreg/compiler/intrinsics/string/TestStringLatin1IndexOfChar.java

```diff
@@ -0,0 +1,84 @@
+/*
```

Comment:
Missing copyright+GPL header in new test. See other tests fro example.

Reply:
Thanks I have added this now